CONTRIBUTING.md

```diff
@@ -1,6 +1,32 @@
-# Web World Wind Design and Coding Guidelines
+# Contributing to Web WorldWind
 
-## General
+The following are a set of guidelines for contributing to the NASA Web WorldWind project. These are mostly guidelines, not
+rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table of Contents
+
+[Asking Questions](#asking-questions)
+
+[Design and Coding Guidelines](#design-and-coding-guidelines)
+
+[Contributing](#contributing)
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting New Features](#suggesting-new-features)
+- [Pull Requests](#pull-requests)
+
+## Asking Questions
```

Comment:
From my understanding, the SUPPORT.md file will help users find resources to the project. For now, I think it would be enough to include these resources directly in the CONTRIBUTING.md file, since I believe it will only direct the user to our website and the community forum. Once the team has decided on what public channels we will be using (FAQs, Slack, etc.), I think then we could add the SUPPORT.md file.